Context/Users/Storage/Context/Users/User.ts

```diff
@@ -34,6 +34,7 @@ export namespace User {
 		return (({ password, permissions, twoFactor, ...user }) => ({
 			...user,
 			permissions: flagly.Flags.stringify(permissions[context.application] ?? {}),
+			...("twoFactor" in user && user.twoFactor ? { twoFactor: true } : {}),
```

Comment:
maybe set this explicitly to false?. with the changes we discussed in slack undefined could mean that we are not allowed to see it.

Reply:
\+